### PR TITLE
update statadvisors with new members @lcolladotor @Paula-Moraga

### PR DESCRIPTION
**Pull request by mpadge**

### CODE REVIEW

**Review by maelle — APPROVED** [review]
great!

File: data/team/statadvisors.json [hunk 1]
```diff
@@ -40,4 +40,26 @@
...
+   "roles": [
+      "advisor"
+   ],
+   "twitter": "Paula_Moraga_",
+   "home": "http://www.paulamoraga.com/"
+},
+{
+   "name": "Leonardo Collado-Torres",
+   "affiliation": "Lieber Institute for Brain Development",
+   "url": "http://lcolladotor.github.io",
```
Comment (maelle) [inline data/team/statadvisors.json:58]:
```suggestion
   "url": "https://lcolladotor.github.io",
```

File: data/team/statadvisors.json [hunk 2]
```diff
@@ -40,4 +40,26 @@
    ],
    "twitter": "topepos",
    "home": "http://appliedpredictivemodeling.com/"
+},
+{
+   "name": "Paula Moraga",
+   "affiliation": "King Abdullah University of Science and Technology",
+   "url": "http://www.paulamoraga.com/",
```
Comment (maelle) [inline data/team/statadvisors.json:47]:
```suggestion
   "url": "https://www.paulamoraga.com/",
```

File: data/team/statadvisors.json [hunk 3]
```diff
@@ -40,4 +40,26 @@
...
+{
+   "name": "Leonardo Collado-Torres",
+   "affiliation": "Lieber Institute for Brain Development",
+   "url": "http://lcolladotor.github.io",
+   "image": "/img/team/leo-torres.png",
+   "roles": [
+      "advisor"
+   ],
+   "twitter": "lcolladotor",
+   "home": "http://lcolladotor.github.io"
```
Comment (maelle) [inline data/team/statadvisors.json:64]:
```suggestion
   "home": "https://lcolladotor.github.io"
```

File: data/team/statadvisors.json [hunk 4]
```diff
@@ -40,4 +40,26 @@
    ],
    "twitter": "topepos",
    "home": "http://appliedpredictivemodeling.com/"
```
Comment (maelle) [inline data/team/statadvisors.json:42]:
```suggestion
   "home": "https://appliedpredictivemodeling.com/"
```

File: data/team/statadvisors.json (hunk 1 above)
Comment (mpadge) [reply data/team/statadvisors.json:58]: Yeah, you're right. I got the link from his own site, but that hasn't been updated. Thanks!

**Pull request merged**